src/mpyl/project.py

```diff
@@ -349,9 +349,9 @@ def from_config(values: Optional[dict]):
         if not values:
             return None
         return TraefikAdditionalRoute(
-            name=values.get("name", {}),
-            middlewares=values.get("middlewares", {}),
-            entrypoints=values.get("entrypoints", {}),
+            name=values.get("name", ""),
```

Comment:
This is not related to the task, just fixing some wrong types that I noticed. Boyscout rule I guess